etc/system-lock/lock.sh

```diff
@@ -64,7 +64,7 @@ fi
 sudo chmod +x /etc/rc.local
 fi
 
-if [ $is_success = true ]; then
-sudo reboot
-fi
+#if [ $is_success = true ]; then
+#sudo reboot
```

Comment:
I think we can move the reboot command to the script (or something) calling it, or add a wrapper function to "freeze and reboot", to let the freeze function be standalone and make that testable.